internal/gitstream/assign.go

```diff
@@ -49,28 +51,24 @@ func (a *Assign) Run(ctx context.Context) error {
 	return nil
 }
 
-func (a *Assign) getOwnersContent(ctx context.Context) (*Owners, error) {
+func (a *Assign) getOwnersContent() (*Owners, error) {
 
-	content, _, _, err := a.GC.Repositories.GetContents(ctx, a.RepoName.Owner, a.RepoName.Repo, ownersFile, nil)
-	if err != nil || content == nil { // `content` can be nil if the filePath refers to a directory
-		return nil, fmt.Errorf("could not get %s's content from %s/%s: %v", ownersFile, a.RepoName.Owner, a.RepoName.Repo, err)
-	}
-	data, err := content.GetContent()
+	data, err := a.FileSystemHelper.ReadFile(ownersFile)
 	if err != nil {
-		return nil, fmt.Errorf("%s's data is invalid: %v", ownersFile, err)
+		return nil, fmt.Errorf("could not read file %s: %v", ownersFile, err)
 	}
 
 	var owners Owners
-	if err := yaml.Unmarshal([]byte(data), &owners); err != nil {
-		return nil, fmt.Errorf("could not unmarshal %s as a yaml: %v", ownersFile, err)
+	if err := yaml.Unmarshal(data, &owners); err != nil {
+		return nil, fmt.Errorf("could not unmarshal file as a yaml: %v", err)
 	}
 
 	return &owners, nil
 }
 
 func (a *Assign) assignIssues(ctx context.Context) error {
 
-	owners, err := a.getOwnersContent(ctx)
+	owners, err := a.getOwnersContent()
 	if err != nil {
 		return fmt.Errorf("could not get owners content: %v", err)
 	}
```

Comment:
Probably unrelated to this very PR, but should the whole business with `OWNERS` be optional? GitStream might be useful to projects that don't use it.

Reply:
I think it can be a good idea but can we postponed this discussion to a later time if we start hitting projects with no OWNERS in it? Seems pretty low priority for now. 

I don't mind we open an issue for it to keep track of it.

Reply:
Created https://github.com/qbarrand/gitstream/issues/51.